test/gateway/pollingInterval.js

```diff
@@ -710,16 +710,6 @@ test('Polling schemas (should properly regenerate the schema when a downstream s
   })
   t.teardown(() => clock.uninstall())
   const oldSchema = `
-    directive @extends on INTERFACE | OBJECT
```

Comment:
It doesn't make sense for a service to have this defined in its schema. This is federation metadata.